Changelog.md

```diff
@@ -1,10 +1,10 @@
 ### 0.8.28 (unreleased)
 
 Language Features:
+ * Transient storage value type state variables are now fully supported.
```

Comment:
```suggestion
 * Transient storage value type state variables are now fully supported (code generation included).
```

Reply:
Well, that's implied by "fully supported", isn't it?

If that's unclear, the blog post would probably be a better place to expand on it.

Reply:
Though, as an aside, the wording is a bit awkward here. This would sound more natural:

```suggestion
 * Transient storage state variables of value types are now fully supported.
```